### PR TITLE
Update to latest model

### LINKED ISSUE #197: Update to new model everywhere
A new model has been trained.

Update all mentions of the `20220825` model to the new `20230808` model.

Includes: all the model cards, config files, default arguments, in the public S3 folder

### PR DESCRIPTION
**Pull request by lizgzil**
Fixes #197 and #212 

- Update references to the old model (20220825) to the new model (20230808)
- Added the new model to a public zipped folder. This is currently in `ojd_daps_skills_data_new.zip` since it will break things if it replaces `ojd_daps_skills_data.zip` just yet
- Add info to model cards
- Optimize bertvectorizer

To do:
- [ ] Rename the old public zipped file to `ojd_daps_skills_data_old.zip` and change `ojd_daps_skills_data_new.zip` to `ojd_daps_skills_data.zip`. DO THIS AFTER THIS PR IS MERGED?

## 

**Important.** I have created a new public S3 zipped file (s3://open-jobs-indicators/escoe_extension/ojd_daps_skills_data_new.zip) which has the new model (20230808) rather than the old one (20220825).

For the meantime, I have kept `s3://open-jobs-indicators/escoe_extension/ojd_daps_skills_data.zip` as it is (i.e. with the old model). This is because whilst this code is still in a PR, dev won't work with the new zipped file (it will try to look for the 20220825 model but it won't exist). 

This is how I updated the zipped file:
```
# Move over the new model
aws s3 cp --recursive s3://open-jobs-lake/escoe_extension/outputs/models/ner_model/20230808/ s3://open-jobs-indicators/escoe_extension/outputs/models/ner_model/20230808/

# Delete the old model
aws s3 rm s3://open-jobs-indicators/escoe_extension/outputs/models/ner_model/[20220825/](https://s3.console.aws.amazon.com/s3/buckets/open-jobs-indicators?region=eu-west-1&prefix=escoe_extension/outputs/models/ner_model/20220825/&showversions=false) --recursive

# Download the whole folder locally
aws s3 cp --recursive s3://open-jobs-indicators/escoe_extension/outputs/ ojd_daps_skills_data/outputs/

# zip
zip -r ojd_daps_skills_data.zip ojd_daps_skills_data/

# Upload the zipped file
aws s3 cp ojd_daps_skills_data.zip  s3://open-jobs-indicators/escoe_extension/ojd_daps_skills_data_new.zip
```

---

Thanks for contributing to Nesta's Skills Extractor Library 🙏!

If you have suggested changes to _code_ anywhere outside of the ExtractSkills class, please consult the checklist below.

Checklist ✔️🐍:

- [ ] I have refactored my code out from `notebooks/`
- [ ] I have checked the code runs
- [ ] I have tested the code
- [ ] I have run `pre-commit` and addressed any issues not automatically fixed
- [ ] I have merged any new changes from `dev`
- [ ] I have documented the code
  - [ ] Major functions have docstrings
  - [ ] Appropriate information has been added to `README`s
- [ ] I have explained the feature in this PR or (better) in `output/reports/`
- [ ] I have requested a code review

If you have suggested changes to _documentation_ (and/or the ExtractSkills class), please **ALSO** consult the checklist below.

Documentation Checklist ✔️📚:

- [ ] I have run `make html` in `docs`
- [ ] I have manually reviewed the `docs/build/*.html` files locally to ensure they have formatted correctly
- [ ] I have pushed both relevant files AND their corresponding `docs/build/*.html` files

### CODE REVIEW

**Comment by lizgzil** [comment]
@india-kerle - no rush on this - but I've tried to update everything with the new model details. Hope I haven't missed something?

File: ojd_daps_skills/pipeline/skill_ner/README.md [hunk 1]
```diff
@@ -79,9 +79,13 @@ For the labelling done at the end of June 2022, we labelled the chunk of 400 job
 
 The outputs of this labelled are stored in `s3://open-jobs-lake/escoe_extension/outputs/skill_span_labels/`.
 
-### Merging labelled files
+## Label data using Prodigy
 
-Since multiple people labelled files from different locations, we merge the labelled data using the following command:
+We labelled another batch of job adverts using [Prodigy](https://prodi.gy/). This was to avail of their active learning capabilities. Details of how we labelled job adverts this way are given in [the Prodigy labelling README](./ojd_daps_skills/ojd_daps_skills/pipeline/skill_ner/prodigy/README.md).
```
Comment (india-kerle) [inline ojd_daps_skills/pipeline/skill_ner/README.md:84]:
```suggestion
We labelled another batch of job adverts using [Prodigy](https://prodi.gy/). This was to make use of their active learning capabilities. Details of how we labelled job adverts this way are given in [the Prodigy labelling README](./ojd_daps_skills/ojd_daps_skills/pipeline/skill_ner/prodigy/README.md).
```

File: outputs/reports/skills_extraction.md [hunk 2]
```diff
@@ -16,11 +16,11 @@ This process means we can extract skills from thousands of job adverts and analy
 
 ## Labelling data
 
-To train the NER model we needed labelled data. First we created a random sample of job adverts and got them into a form needed for labelling using [Label Studio](https://labelstud.io/). More about this labelling process can be found in the `skill_ner` pipeline [README.md](./ojd_daps_skills/ojd_daps_skills/pipeline/skill_ner/README.md).
+To train the NER model we needed labelled data. First we created a random sample of job adverts and got them into a form needed for labelling using [Label Studio](https://labelstud.io/), we then did a second batch of labelled using [Prodigy](https://prodi.gy/). More about this labelling process can be found in the `skill_ner` pipeline [README.md](./ojd_daps_skills/ojd_daps_skills/pipeline/skill_ner/README.md).
```
Comment (india-kerle) [inline outputs/reports/skills_extraction.md:19]:
```suggestion
To train the NER model we needed labelled data. First we created a random sample of job adverts and got them into a form needed for labelling using [Label Studio](https://labelstud.io/). We then did a second batch of labelled using [Prodigy](https://prodi.gy/). More about this labelling process can be found in the `skill_ner` pipeline [README.md](./ojd_daps_skills/ojd_daps_skills/pipeline/skill_ner/README.md).
```

**Review by india-kerle — APPROVED** [review]
Looks great! thanks for these changes :)

**Pull request merged**